packages/firestore/src/api/user_data_converter.ts

```diff
@@ -556,10 +556,7 @@ export class UserDataConverter {
    *
    * @return The parsed value
    */
-  private parseScalarValue(
-    value: AnyJs,
-    context: ParseContext
-  ): FieldValue {
+  private parseScalarValue(value: AnyJs, context: ParseContext): FieldValue {
```

Comment:
@jshcrowthe this keeps on happening where another modules code is touched because of formatting - this really isn't good for PR's.